testing/procedural/build.gradle.kts

```diff
@@ -1,8 +1,21 @@
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+
 plugins {
     id("yatagan.base-module")
     application
 }
 
+kotlin {
```

Comment:
Can be extracted into a template, something like `yatagan.test-only-module`, I see it's repeated at least two times.